src/ucp/core/ucp_context.c

```diff
@@ -638,6 +638,51 @@ const char* ucp_tl_bitmap_str(ucp_context_h context, uint64_t tl_bitmap,
     return str;
 }
 
+static char* ucp_feature_flag_str(unsigned feature_flag)
+{
+    switch (feature_flag) {
+        case UCP_FEATURE_TAG:
+            return "UCP_FEATURE_TAG";
+        case UCP_FEATURE_RMA:
+            return "UCP_FEATURE_RMA";
+        case UCP_FEATURE_AMO32:
+            return "UCP_FEATURE_AMO32";
+        case UCP_FEATURE_AMO64:
+            return "UCP_FEATURE_AMO64";
+        case UCP_FEATURE_WAKEUP:
+            return "UCP_FEATURE_WAKEUP";
+        case UCP_FEATURE_STREAM:
+            return "UCP_FEATURE_STREAM";
+        default:
+            ucs_error("Unknown feature flag value %u", feature_flag);
+            return NULL;
```

Comment:
maybe return "<invalid feature flag>" instead of printing error

Reply:
then may be `ucs_fatal`?